drivers/platform/freeRTOS/freertos_mutex.c

```diff
@@ -40,6 +40,9 @@
 #include <FreeRTOS.h>
 #include "no_os_mutex.h"
 #include "semphr.h"
+#include "queue.h"
+
+#define freeRTOS_WAIT_TIME 0
```

Comment:
I think this should be portMAX_DELAY and you could use it directly at line 73
https://www.freertos.org/a00122.html

Reply:
Apparently 0 really means 0 timeout and it used for "semaphore polling" and portMAX_DELAY can either mean the maximum wait time (0xFFFFFFFF on some platforms) or it can mean infinite wait time